MapboxDirectionsTests/RouteStepTests.swift

```diff
@@ -157,5 +157,6 @@ class RouteStepTests: XCTestCase {
         XCTAssertEqual(unarchivedStep.transportType, step.transportType)
         XCTAssertEqual(unarchivedStep.destinations ?? [], step.destinations ?? [])
         XCTAssertEqual(unarchivedStep.instructionsSpokenAlongStep ?? [], step.instructionsSpokenAlongStep ?? [])
+        XCTAssertEqual(unarchivedStep.instructionsDisplayedAlongStep ?? [], step.instructionsDisplayedAlongStep ?? [])
```

Comment:
Besides coding tests, is there anything else we can test about this new feature? I would’ve expected a test fixture to be updated as part of adding tests.